.circleci/config.yml

```diff
@@ -0,0 +1,71 @@
+orbs:
+  architect: giantswarm/architect@0.4.3
+
+version: 2.1
+jobs:
+  build:
+    machine: true
+    steps:
+    - checkout
+
+    - run: |
+        wget -q $(curl -sS  https://api.github.com/repos/giantswarm/architect/releases/latest | grep browser_download_url | head -n 1 | cut -d '"' -f 4)
+        chmod +x ./architect
+        ./architect version
+
+    - run: |
+        date +"%Y" > /tmp/year
+        date +"%m" > /tmp/month
+        date +"%d" > /tmp/day
+    - restore_cache:
+        keys:
+        - go-cache-v1-{{ checksum "/tmp/year" }}-{{ checksum "/tmp/month" }}-{{ checksum "/tmp/day" }}
+        - go-cache-v1-{{ checksum "/tmp/year" }}-{{ checksum "/tmp/month" }}
+        - go-cache-v1-{{ checksum "/tmp/year" }}
+
+    - run: ./architect build
+
+    - store_test_results:
+        path: /tmp/results
+
+    - persist_to_workspace:
+        root: .
+        paths:
+        - ./template-operator
+
+    - save_cache:
+        key: go-cache-v1-{{ checksum "/tmp/year" }}-{{ checksum "/tmp/month" }}-{{ checksum "/tmp/day" }}
+        paths:
+        - /tmp/go/cache
+
+workflows:
+  build:
+    jobs:
+      - build:
+          filters:
+            tags:
+              only: /^v.*/
+
```

Comment:
Definitely not a blocker but should it also push to Aliyun?

Reply:
Aliyun push is included.